src/tiledb/cloud/utilities/wheel.py

```diff
@@ -180,5 +234,8 @@ def install_wheel(
         print(res.stderr)
 
     # Modify sys.path if the wheel was installed with --user
-    if not installer.in_venv and USER_SITE not in sys.path:
+    from site import USER_SITE
+
+    if not installer.in_venv:
```

Comment:
@gspowley I made this last if statement, less stringent since we want to ensure even if the user site is in the sys.path that it sits at index 0

Reply:
That makes sense. The goal was to prevent the path from growing too large when running `install_wheel` multiple times. We could use something like this:

```python
    if not installer.in_venv and sys.path[0] != USER_SITE:
```

It's a rare edge case though. What you have is good.

Reply:
going to add